.github/workflows/tests-main.yaml

```diff
@@ -69,7 +69,7 @@ jobs:
         name: 'Run E2E Tests'
         run: "./tools/e2e_test_job.sh"
         env:
-          ENVIRONMENT_NAME: "ci-main"
+          ENVIRONMENT_NAME: "ci-daily"
```

Comment:
This also runs on `push` events right? We should probably think of a better name that incapsulates periodic and continuous builds because `daily` isn't fully accurate.

Reply:
I have to change the name from 'ci-main' to something else, or else it will break. Google Cloud can't fully delete all the resources with 'ci-main' in the name - some networking components take a day or two to fully disappear.

What about 'ci-branch-main'?